Tensile/Contractions.py

```diff
@@ -316,8 +316,9 @@ def CompoundPredicates(cls, state, problemType):
         if "KernelLanguage" in state:
             rv += [cls("KernelLanguageCompatible", value=state["KernelLanguage"])]
 
-        if 'GlobalSplitU' in state and state['GlobalSplitU'] > 1:
-            rv += [cls("DeterministicMode", value = False)]
+        if ('GlobalSplitU' in state) and (state['GlobalSplitU'] > 1):
+            if ('_GlobalAccumulation' not in state) or (state['_GlobalAccumulation'] != 2):
+                rv += [cls("DeterministicMode", value = False)]
```

Comment:
Does the _GlobalAccumulation 2 algo use atomics ? If yes then DeterministicMode predicate for 'false' should be included.